lib/portlayer/storage/storage.go

```diff
@@ -58,11 +59,19 @@ func create(ctx context.Context, session *session.Session, pool *object.Resource
 
 	mngr := view.NewManager(session.Vim25())
 
+	op := trace.FromContext(ctx, "storage component initialization")
+
 	// Create view of VirtualMachine objects under the VCH's resource pool
-	Config.ContainerView, err = mngr.CreateContainerView(ctx, pool.Reference(), []string{"VirtualMachine"}, true)
+	Config.ContainerView, err = mngr.CreateContainerView(op, pool.Reference(), []string{"VirtualMachine"}, true)
```

Comment:
Do we have a constant we could reference instead of `VirtualMachine`?

Reply:
Not that I can find. @dougm?